src/main/java/lotto/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,71 @@
+package lotto.domain.lotto;
+
+import java.util.List;
+
+public class Lotto {
+    private static final String NUMBERS_SIZE_ERROR_MESSAGE = "로또 번호는 6개이어야 합니다.";
+    private static final String NUMBERS_DUPLICATE_ERROR_MESSAGE = "로또 번호에 중복이 있으면 안됩니다.";
+    private static final String NUMBER_RANGE_ERROR_MESSAGE = "로또 번호는 1~45 의 숫자입니다.";
```

Comment:
상수로 메시지를 관리하는 것은 좋아보입니다.
다만, 저는 로또 번호의 범위에 대한 정보가 `String`과 `int`에 중복되어 존재하는 것 역시 일종의 코드 중복이라고 생각해요.
```suggestion
    private static final String NUMBER_RANGE_ERROR_MESSAGE = String.format("로또 번호는 %d~%d 의 숫자입니다.",LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER);
```

Reply:
오,, 좋은 방법인 것 같네요 하나 배워갑니다!